tools/releaseBuild/azureDevOps/vpackRelease.yml

```diff
@@ -45,6 +45,12 @@ stages:
         ReleaseTagVar: $(ReleaseTagVar)
         CreateJson: yes
         UseJson: no
+        
+    - powershell: |
+        if($env:RELEASETAGVAR -match '-') {
+            throw "Don't release a preview build without coordinating with Windows Engineering Build Tools Team"
```

Comment:
Will this fail the release pipeline? It may be better to not fail the pipeline, but just make it a non-op?

Reply:
We should fail the pipeline where the action fails.